addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -1368,14 +1368,16 @@ function makeActionManager(env) {
     function pushState(controller) {
         const newState = {};
         const action = controller.action;
+        const viewType = controller.props.type;
```

Comment:
this changes looks fine, but it should come with an explanation of the motivation, and a test